source/http/httpsource.go

```diff
@@ -392,6 +392,9 @@ func (hs *httpSourceHandler) Snapshot(ctx context.Context, g session.Group) (cac
 	if err != nil {
 		return nil, err
 	}
+	defer func() {
+		_ = resp.Body.Close()
```

Comment:
Can we not do `defer resp.Body.Close()`? Or is the current form required by the linter to explicitly discard the error? :thinking:

Reply:
Yeah it's fine to also use `defer resp.Body.Close()`

Reply:
Would rather use that then if we can :+1: